extract_otp_tokens.py

```diff
@@ -22,6 +22,20 @@ def parse_bool(value):
     return value.lower() in {'y', 'yes', 'true'}
 
 
+def adb_list_dir(path):
+    process = subprocess.Popen(
+            args=['adb', 'shell', f'su -c ls', path],
```

Comment:
I've had quoting issues when using `adb shell` on some phones. Using `['adb', 'shell', f'su -c ls {shlex.quote(path)}']` seems to work better.

Reply:
Done, also there's some magic you did to avoid process.communicate() and make it faster, but I have no idea what's going on there.
I'm a bit new to this GH thing, do you not have the ability to change stuff in the PR? It'd probably be easier if I added you as a collaborator.